lisa/wa.py

```diff
@@ -552,3 +553,61 @@ def __init__(self, wa_output, trace_to_df, **kwargs):
     def _get_artifact_df(self, path):
         trace = Trace(path, **self._trace_kwargs)
         return self._trace_to_df(trace)
+
+class WAJankbenchCollector(WAArtifactCollectorBase):
+    """
+    WA collector for the jankbench frame timings.
+
+    The collector framework will return a single :class:`pandas.DataFrame` with the
+    results from every jankbench job in WACollector database format (i.e. the returned dataframe
+    is arranged such that each reported metric is separated as a separate row). The metrics
+    reported are:
+
+    . ``total_duration``: Time in milliseconds to complete the frame
+
+    . ``jank_frame``: Boolean indicator of missed frame deadline. ``1`` is a Jank frame, ``0`` is not.
+
+    . ``name``: Subtest name, provided by the Jankbench app
+
+    . ``frame_id``: monotonically increasing frame number, starts from ``1`` for each subtest iteration.
+
+    An example plotter matching the old-style output can be found in the jupyter notebook
+    working directory at :file:`ipynb/wltests/WAOutput-JankbenchDemo.ipynb`
+
+    If you have existing code expecting a more direct translation of the original sqlite database
+    format, you can massage the collected dataframe back into a closer resemblance to the
+    original source database with this sequence of pandas operations::
+
+        collector = wa.get_collector('jankbench')
```

Comment:
it's a bit confusing as `wa` is a module which does not have any `get_collector` function. Would be better to stay closer to the other docstring example:
```
        wa_output = WAOutput('wa/output/path')
        df = wa_output['jankbench'].df
        ...
```